src/behaviors/focusZone.ts

```diff
@@ -587,6 +587,10 @@ export function focusZone(container: HTMLElement, settings?: FocusZoneSettings):
       return 0
     }
 
+    if (currentFocusedElement === container) {
+      return -1
+    }
```

Comment:
Does this work in a situation where `container` is already focusable?  Technically we would want to return `0` in that case, right?

Reply:
Fixed in https://github.com/primer/components/pull/1270/commits/40fac046d9d09f4bf9b548f0f6b1c50167959052